search/includes/classes/class-versioning.php

```diff
@@ -741,7 +745,17 @@ public function replicate_queued_objects_to_other_versions( $queued_objects ) {
 					// Override the index version in the options
 					$options['index_version'] = $version['number'];
 
-					\Automattic\VIP\Search\Search::instance()->queue->queue_object( $object_id, $object_type, $options );
+					/**
+					 * Filter do determine the priority of the replication job
+					 *
+					 * @param int $priority         Priority
+					 * @param int $object_id        Object ID
+					 * @param string $object_type   Object type
+					 * @return int                  Priority
+					 */
+					$options['priority'] = apply_filters( 'ep_versioning_reindex_priority', self::VERSIONING_JOB_DEFAULT_PRIORITY, $object_id, $object_type );
```

Comment:
I think we should name this filter with `vip_` prefix as this is not ElasticPress feature if I understand correctly.
```suggestion
					$options['priority'] = apply_filters( 'vip_inactive_version_reindex_priority', self::INACTIVE_VERSION_JOB_DEFAULT_PRIORITY, $object_id, $object_type );
```

Reply:
Fixed in 09af12f04